database/seeds/UserTableSeeder.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class UserTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $user = new App\Models\User([
```

Comment:
You can use UserFactory to seed into your database, instead of hardcoding here again.